internal/pkg/service/buffer/api/service/service.go

```diff
@@ -81,25 +82,10 @@ func (s *service) CreateReceiver(d dependencies.ForProjectRequest, payload *buff
 	receiver.Exports = exports
 
 	// Create Storage files for exports
-	mutex := &sync.RWMutex{}
-	files := make(map[key.ExportKey]*storageapi.File)
-	wg := client.NewWaitGroup(ctx, d.StorageAPIClient())
-	for _, export := range receiver.Exports {
-		expKey := export.ExportKey
-		wg.Send(
-			storageapi.CreateFileResourceRequest(&storageapi.File{
-				Name:     export.Name,
-				IsSliced: true,
-			}).WithOnSuccess(func(ctx context.Context, sender client.Sender, result *storageapi.File) error {
-				mutex.Lock()
-				files[expKey] = result
-				mutex.Unlock()
-				return nil
-			}),
-		)
-	}
-	if err = wg.Wait(); err != nil {
-		return nil, err
+	m := file.NewManager(d.StorageAPIClient())
```

Comment:
Proč to není v `setupExport`?

Reply:
taky vidím v `store.CreateExport`, že je tam ten file zvlášť. Když je vždycky associated s tím exportem, proč není v tom exportu? @michaljurecko nevím kde se tyhle změny udělaly, ale dost mě to mate

Reply:
sedím u toho už hodinu a půl a nenapsal jsem jedniný řádek kódu. vůbec nechápu, co vlastně tady mám dokončovat.

Reply:
No nie je to tam preto, lebo sme povedali, ze vsetky operacie ohladom suborov dame bokom.

Treba dokoncit `UploadSlicedFileManifest`, skontrolovat ci nechyba nejaky bod podla zadania.

Reply:
Když je to bokem, tak proč se teda vytváří zároveň s exportama?

Reply:
Cielom bolo spravit service lepsie citatelnou aby sapajala iba dokopy viacere ine balicky, kde je logika.

Fakt teraz nie je vhodná doba, riesit nejaky nepodstatny detail,
mozes sa prosim zmieri s tym, ze je to tu a pokracovat?

Reply:
nejde mi o to něco měnit, ale nemůžu s tím pokračovat, když nechápu ten kód

Reply:
Lebo to potrebujeme cim skor dokoncit, Jakub je chory a mam este dost inej roboty ako teraz riesit tento detail.

Pryvy krok moze byt, dokoncit toto:
![image](https://user-images.githubusercontent.com/19371734/207600514-e676e23a-3123-439c-a2bd-a6757226c0c6.png)

`w, err` -> `_, err`
`return err`

Reply:
Mozem si s tebou kludne na tym zavolat, len teraz idem na iny call (office hours).
A zatial si sa nespytal nic konkretne :slightly_smiling_face:, okrem toho, ze preco sa ten kod presunul.

Reply:
Ak by si nevedel uz fakt co spravit, tak mozes zatial dopista chybajuce unit testy podla zadania a na `sliceNumberToFilename`.

Reply:
posterity: měli jsme call, udělá se list uploaded slices + dodělají se unit testy na store a `sliceNumberToFilename`